resources/lang/en/patrols/new_cat.json

```diff
@@ -4327,5 +4327,393 @@
+                ],
+                "outsider_rep": 0
+            },
+            {
+                "text": "A loner? How interesting. p_l approaches the outsider but, before {PRONOUN/p_l/subject} can even meow out a greeting, the loner takes a swipe at {PRONOUN/p_l/object} and bolts away.",
+                "exp": 0,
+                "weight": 10,
+                "new_cat": [
+                    ["meeting", "loner", "age:adolescent"]
+                ],
+                "injury": [
+                    {
+                        "cats": ["patrol"],
+                        "injuries": ["claw-wound", "torn pelt", "torn ear", "minor_injury"],
+                        "scars": ["ONE", "TWO", "THREE", "RIGHTEAR", "LEFTEAR", "RIGHTBLIND", "LEFTBLIND", "BRIDGE", "FACE", "SNOUT", "CHEEK", "THROAT"]
+                    }
+                ],
+                "history_text": {
+                    "scar": "m_c was scarred after being attacked by a startled loner.",
+                    "death": "m_c died from injuries {PRONOUN/p_l/subject} sustained after being attacked by a startled loner."
+                },
+                "outsider_rep": 0
+            },
+            {
+                "text": "What in StarClan's name is a <i>kittypet</i> doing here? c_n's territory is no place for such a pampered creature, so p_l approaches the other young cat, intending to send {PRONOUN/n_c:0/object} back to wherever {PRONOUN/n_c:0/subject} came from. However, p_l's presence clearly intimidates the kittypet, who takes a swipe at p_l and bolts away the moment {PRONOUN/n_c:0/subject} {VERB/n_c:0/see/sees} {PRONOUN/p_l/object}.",
+                "exp": 0,
+                "weight": 5,
+                "new_cat": [
+                    ["meeting", "kittypet", "age:adolescent"]
+                ],
+                "injury": [
+                    {
+                        "cats": ["patrol"],
+                        "injuries": ["claw-wound", "torn pelt", "torn ear", "minor_injury"],
+                        "scars": ["ONE", "TWO", "THREE", "RIGHTEAR", "LEFTEAR", "RIGHTBLIND", "LEFTBLIND", "BRIDGE", "FACE", "SNOUT", "CHEEK", "THROAT"]
+                    }
+                ],
+                "history_text": {
+                    "scar": "m_c was scarred after being attacked by a startled kittypet.",
+                    "death": "m_c died from injuries {PRONOUN/p_l/subject} sustained after being attacked by a startled kittypet."
+                },
+                "outsider_rep": 0
+            },
+            {
+                "text": "p_l never planned for things to happen like this but, when {PRONOUN/p_l/subject} {VERB/p_l/approach/approaches} the loner and begins talking with {PRONOUN/n_c:0/object}, {PRONOUN/p_l/subject} can't help but feel intrigued by the life the other young cat leads. Perhaps it would do {PRONOUN/p_l/object} good to experience what the world has to offer outside of c_n...",
```

Comment:
```suggestion
                "text": "p_l never planned for things to happen like this, but when {PRONOUN/p_l/subject} {VERB/p_l/approach/approaches} the loner and begins talking with {PRONOUN/n_c:0/object}, {PRONOUN/p_l/subject} can't help but feel intrigued by the life the other young cat leads. Perhaps it would do {PRONOUN/p_l/object} good to experience what the world has to offer outside of c_n...",
```